src/translations/constants.ts

```diff
@@ -0,0 +1,17 @@
+import {messages_ru} from './ru.js'
+import {messages_en} from './en.js'
+
+ const LANGS = [
+    {
+        value: 'en',
```

Comment:
let's rename 'value' to 'code'
https://en.wikipedia.org/wiki/ISO_639-1

Do it in this cons and everywhere in helper functions,
e.g. getCurrentLanguageValue -> getCurrentLanguageCode

And let's do not use short 'lang', 'LANG'-> language, LANGUAGES everywhere